### PR TITLE
Fixed getImageData warning

### PR DESCRIPTION
**Pull request by alexanderby**
- Use `willReadFrequently` option.

### CODE REVIEW

File: src/inject/dynamic-theme/image.ts [hunk 1]
```diff
@@ -124,7 +124,7 @@ function analyzeImage(image: HTMLImageElement) {
     context.clearRect(0, 0, width, height);
 
     context.drawImage(image, 0, 0, naturalWidth, naturalHeight, 0, 0, width, height);
-    const imageData = context.getImageData(0, 0, width, height);
+    const imageData = context.getImageData(0, 0, width, height, {willReadFrequently: true} as any);
```
Comment (Gusted) [inline src/inject/dynamic-theme/image.ts:127]: You need to add `{willReadFrequently: true}` to the `getContext` function. But this is optimistic, as there aren't that many sites that will likely see a lot of images that need toe be analyzed. Not sure what browsers does with this option, but it shouldn't negatively impact other sites with just a few images.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/inject/dynamic-theme/image.ts (hunk 1 above)
Comment (alexanderby) [reply src/inject/dynamic-theme/image.ts:127]: I was seeing this warning on almost every website, including Gmail, Linkedin...

Comment (Gusted) [reply src/inject/dynamic-theme/image.ts:127]: Might be me, but I don't visit "the major" sites from day-to-day 😅.

Comment (alexanderby) [reply src/inject/dynamic-theme/image.ts:127]: Ah, you are right, it's the wrong place to put the flag 🤦

**Review by Gusted — APPROVED** [review]

**Pull request merged**